server/modules/modules.go

```diff
@@ -48,5 +49,96 @@ func Auth(message *messageService.Message, conn *websocket.Conn) {
+}
+
+func ChangePass(message *messageService.Message, messageType int, conn *websocket.Conn) {
+	ok, err := settingService.ChangePass(*message)
+	if err != nil {
+		loger.Log.Errorf(" Error has occurred : ", err)
+		byteError, _ := json.Marshal(changePassError) // this for UI
+		conn.WriteMessage(messageType, byteError)     // this for UI
+		return
+	}
+	if !ok {
+		loger.Log.Warnf(" Password has not changed. Please try again")
+		byteError, _ := json.Marshal(changePassError) // this for UI
+		conn.WriteMessage(messageType, byteError)     // this for UI
+		return
+	}
+	byteOk, _ := json.Marshal(changePassOk) // this for UI
+	conn.WriteMessage(messageType, byteOk)  // this for UI
+	return
+}
+
+func ChangeNickName(message *messageService.Message, messageType int, conn *websocket.Conn) {
+	ok, err := settingService.ChangeNickName(*message)
+	if err != nil {
+		loger.Log.Errorf(" Error has occurred : ", err)
+		byteError, _ := json.Marshal(changeNicknameError) // this for UI
+		conn.WriteMessage(messageType, byteError)         // this for UI
+		return
+	}
+	if !ok {
+		loger.Log.Warnf(" Nickname has not changed. Please try again")
+		byteError, _ := json.Marshal(changeNicknameError) // this for UI
+		conn.WriteMessage(messageType, byteError)         // this for UI
+		return
+	}
+	byteOk, _ := json.Marshal(changeNicknameOk) // this for UI
+	conn.WriteMessage(messageType, byteOk)      // this for UI
+	return
+}
+
+func ChangeBirthday(message *messageService.Message, messageType int, conn *websocket.Conn) {
+	ok, err := settingService.ChangeBirthday(*message)
+	if err != nil {
+		loger.Log.Errorf(" Error has occurred : ", err)
+		byteError, _ := json.Marshal(changeBirthdayError) // this for UI
+		conn.WriteMessage(messageType, byteError)         // this for UI
+		return
+	}
+	if !ok {
+		loger.Log.Warnf(" Birthday has not changed. Please try again")
+		byteError, _ := json.Marshal(changeBirthdayError) // this for UI
+		conn.WriteMessage(messageType, byteError)         // this for UI
+		return
+	}
+	byteOk, _ := json.Marshal(changeBirthdayOk) // this for UI
+	conn.WriteMessage(messageType, byteOk)      // this for UI
+	return
+}
+
+func ChangeAboutUserInfo(message *messageService.Message, messageType int, conn *websocket.Conn) {
+	ok, err := settingService.ChangeAboutUserInfo(*message)
+	if err != nil {
+		loger.Log.Errorf(" Error has occurred : ", err)
+		byteError, _ := json.Marshal(changeUserInfoError) // this for UI
+		conn.WriteMessage(messageType, byteError)         // this for UI
+		return
+	}
+	if !ok {
+		loger.Log.Warnf(" Info about user has not changed. Please try again")
+		byteOk, _ := json.Marshal(changeUserInfoOk) // this for UI
+		conn.WriteMessage(messageType, byteOk)      // this for UI
+		return
+	}
+	return
+}
+
+func BlockUnblockUser(message *messageService.Message, messageType int, conn *websocket.Conn) {
+	ok, err := settingService.BlockUnblockUser(*message)
```

Comment:
Why are you trying to copy value to function if it require pointer? Have you tried to run this code?